src/cache.js

```diff
@@ -63,4 +63,8 @@ async function cachedFetch(url, cache, ttl) {
   }
 }
 
-module.exports = { cachedFetch, expire };
+function resetCallCounter() {
+  callCounter = {};
+}
```

Comment:
I think the proper solution to this will probably be to encapsulate this in a class so every test can instantiate a new cache object (with a clean `callCounter`) but I agree we can work around it for now and punt it to an issue for another day.